### PR TITLE
Mark.viterna/api 5929 forms doc refresh

### PR DESCRIPTION
**Pull request by mviterna**
## Original issue(s)
https://vajira.max.gov/browse/API-5929
## Things to know about this PR

UI approved and tested

### CODE REVIEW

File: modules/va_forms/app/swagger/va_forms/v0/controller_swagger.rb [hunk 1]
```diff
@@ -10,9 +10,8 @@ class ControllerSwagger
           security do
             key :apikey, []
           end
-
-          key :summary, 'All VA Forms'
-          key :description, 'Returns all VA Forms and their last revision date'
+          key :summary, 'Returns all VA Forms and their last revision date'
+          key :description, 'Returns an index of all available VA forms. Optionally, pass a ?query parameter to filter forms by form number or title.'
```
Comment (bschofield-va) [inline modules/va_forms/app/swagger/va_forms/v0/controller_swagger.rb:14]: "pass a ?query parameter" seems a bit odd to me. Why the `?` in the middle of the sentence?

**Comment by bschofield-va** [comment]
Has UX signed signed off on the changes? @darduda?

File: modules/va_forms/app/swagger/va_forms/v0/controller_swagger.rb (hunk 1 above)
Comment (mviterna) [reply modules/va_forms/app/swagger/va_forms/v0/controller_swagger.rb:14]: Yeah, probably don't need it. The question mark should be implied for a query param. I'll remove it

**Review by gpbowman-git — APPROVED** [review]

**Comment by NCHLiberty** [comment]
> Has UX signed signed off on the changes? @darduda?

hi @bschofield-va yes we walked through with Darcy Duda yesterday and incorporated her suggestions. This is approved from Darcy and my points of view.

**Review by darduda — COMMENTED** [review]
We reviewed this during a workshop yesterday and I provided some updates. There's nothing that should hold this back from a UX perspective right now.

**Pull request merged**